concourse/pipelines/linux-image-build.jsonnet

```diff
@@ -385,16 +386,40 @@ local DebianImgPublishJob(image, env, workflow_dir) = imgpublishjob {
   image_prefix: common.debian_image_prefixes[image],
 };
 
-local ImgGroup(name, images) = {
-  name: name,
+local SapWorkloadTestJob(image) = sap_test.saptestjob {
+  image: image,
+};
+
+local imggroup = {
+  local tl = self,
+
+  group_name:: '',
+  images:: [],
+  extra_job_groups:: [],
+  
+  name: self.group_name,
   jobs: [
     'build-' + image
-    for image in images
+    for image in self.images
   ] + [
     'publish-to-%s-%s' % [env, image]
     for env in envs
-    for image in images
-  ],
+    for image in self.images
+  ] + tl.extra_job_groups,
+};
+
+local ImgGroup(name, images) = imggroup {
+  group_name: name,
+  images: images,
+};
+
+local RhelImgGroup(images) = imggroup {
+  group_name: 'rhel',
+  images: images,
+  extra_job_groups: [
+    'sap-workload-test-%s' % [image]
+    for image in std.filter(function(x) std.endsWith(x, '-sap') , images)
+  ]
 };
```

Comment:
please restore this, you just want the rhel jobs to have extra jobs added we can do that with a composition right at the invocation point. i'll show an example of that.